### ISSUE TITLE
Settings Page Improvements

### ISSUE THREAD

**Issue by EugenioPetulla** [deshack/pinit#28]
- [ ] "Advanced" Tab
- [ ] Sidebar link for donate
- [ ] Sidebar Link to @deshack (TODO)
- [ ] SDK Freemius (This could be exploded in a new task and treated later)

I think we can start improving the settings page integrating a "Advanced" tab where put a disabling option for widgets (if you don't use it don't load it at all!) and another one for shortcodes (same story).

**Actually this is the admin settings page:**
![schermata da 2016-02-24 14-15-41](https://cloud.githubusercontent.com/assets/3589467/13402396/5c1e1a8a-df10-11e5-8574-79c89c4bb2ed.png)

**Comment by deshack** [comment]
Also note that in larger screens the sidebar boxes are in a row, not in a column. See screenshot below:

![image](https://cloud.githubusercontent.com/assets/2034213/13402763/0b2ee364-df12-11e5-81d0-1445156516ae.png)

**Comment by EugenioPetulla** [comment]
This is how they suppose to be following the WordPress dashboard rules. That's wordpress' css, not mine. We can consider to provide some workaround by putting some roles into admin.css but maybe it's better to rethink the concept of this page putting some good content into sidebars. :)

**Comment by deshack** [comment]
@EugenioPetulla may I suggest to use white icons instead of black in the _follow us_ block? :)

**Comment by EugenioPetulla** [comment]
@deshack yes, we have to reconsider this because actually are just black icons inverted by a css filter (that's not working in your case). But I'm working on a workaround using jetpack photon CDN that runs that images and I remember there is a "server side filter API" to do this as well.

**Comment by EugenioPetulla** [comment]
No, photon doesn't seems to have this feature so I add a task to this list in order to change the original source image.

https://jetpack.me/support/photon/

**Comment by EugenioPetulla** [comment]
Responsive/Sidear issues fixed by providing special rules in admin.css

**Comment by EugenioPetulla** [comment]
@deshack some of this stuff are actually fixed and ready for the 2.1.1 release.

You can include them into the changelog if you want. Look at this commit: 39bdee0

**Comment by deshack** [comment]
@EugenioPetulla can you update the task list in the description with what has been already done?

**Comment by EugenioPetulla** [comment]
@deshack I already updated it, all the stuffs that are cheked was already done (your link in the sidebar, white social icons and fix for responsive layout. Averything was included in the 2.1.1 relaese! :)

**Comment by EugenioPetulla** [comment]
I have done also the woocommerce stuff with this commit but you didn't included it in the release:

https://github.com/deshack/pinit/tree/feature/granular-pages-option

Maybe you have to merge the code bacause I'm losing the path in this way! 👍
